src/Web/Base/EntityModificationAction.php

```diff
@@ -42,6 +41,12 @@ protected function loadModelAndSave() {
         return $isSaved;
     }
 
+    protected function processResponse(bool $isSaved) {
```

Comment:
except having this method based on a flag, just create `handleFailedOperation` like you did with `processSuccessfulResult` and change return of `loadModelAndSaveOrPrintView` to
```php
return $this->loadModelAndSave() ? 
                    $this->handleSuccessfulOperation() 
                    : $this->handleFailedOperation();
```